default/scripting/species/common/telepathic.macros

```diff
@@ -0,0 +1,16 @@
+TELEPATHIC_DETECTION
+'''     EffectsGroup
+            description = "TELEPATHIC_DETECTION"
+            scope = And [
+                PopulationCenter
+                Not OR [
+                    Species name = Source.Species
+                    Species name = "SP_EXOBOT"
```

Comment:
Ok, decided to remove that condition entirely - which means, Telepathic Detection works on all populated planets regardless of species. This shouldn't be a problem here (on the contrary, excluding planets with the same species as the "detecting" species doesn't make sense), unlike the xenophobic effects, where your own species and exobots of course need to be excluded.

If we want certain species to be excluded from Telepathic Detection, we can still add this in later on, but for simplicities sake I'm going to remove that bit from this PR.